soc/nordic/common/pinctrl_soc.h

```diff
@@ -154,6 +152,20 @@ typedef uint32_t pinctrl_soc_pin_t;
  */
 #define NRF_GET_PIN(pincfg) (((pincfg) >> NRF_PIN_POS) & NRF_PIN_MSK)
 
+/**
+ * @brief Utility macro to obtain port.
+ *
+ * @param pincfg Pin configuration bit field.
+ */
+#define NRF_GET_PORT(pincfg) (NRF_GET_PIN(pincfg) >> 5)
+
+/**
+ * @brief Utility macro to obtain port.
```

Comment:
pin

Reply:
changed to `@brief Utility macro to obtain pin relative to port`